android/app/src/main/java/com/google/codeu/chatme/presenter/CreateConversationPresenter.java

```diff
@@ -46,4 +46,23 @@ public void onComplete(DatabaseError databaseError, DatabaseReference databaseRe
                     }
                 });
     }
+
+    @Override
+    public void addGroupConversation(Conversation conversation) {
```

Comment:
Seems like `addConversation` and `addGroupConversation` can be merged. Before launching the next activity, you could check if the conversation is a group. Am I missing anything?